src/bindings/python/tests/test_graph/test_preprocess.py

```diff
@@ -344,8 +345,12 @@ def test_graph_preprocess_steps(algorithm, color_format1, color_format2, is_fail
         assert "is not convertible to" in str(e.value)
     else:
         function = custom_processor.build()
-        input_data = np.array([[[[1, 2, 3], [4, 5, 6], [7, 8, 9]]]]).astype(np.float32)
-        expected_output = np.array([[[[0, 3, 6], [1, 4, 7], [2, 5, 8]]]]).astype(np.float32)
+        input_data = np.array([[[[1, 2, 3], [4, 5, 6], [7, 8, 9]]], 
+                               [[[1, 2, 3], [4, 5, 6], [7, 8, 9]]], 
+                               [[[1, 2, 3], [4, 5, 6], [7, 8, 9]]]]).astype(np.float32)
+        expected_output = np.array([[[[0, 3, 6], [1, 4, 7], [2, 5, 8]]], 
```

Comment:
Why do we need these changes?

Reply:
The test contains a mistake - input tensor with ColorFormat::RGB means "C" channel should be equal to 3. In this PR I introduce strict logic with "C == 3", so the test started to fail